// @tailus/themer-plugins/dist/palettes/index.js
"use strict";
var __assign = (this && this.__assign) || function () {
    __assign = Object.assign || function(t) {
        for (var s, i = 1, n = arguments.length; i < n; i++) {
            s = arguments[i];
            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
                t[p] = s[p];
        }
        return t;
    };
    return __assign.apply(this, arguments);
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.defaultFeedbackColors = exports.grays = exports.palettes = void 0;
var grays_1 = require("./grays");
var defaultFeedbackColors_1 = require("./defaultFeedbackColors");
var colors_1 = __importDefault(require("tailwindcss/colors"));
exports.palettes = {
    passion: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.red, secondary: colors_1.default.orange, accent: colors_1.default.amber, gray: grays_1.grays.neutral }),
    mystery: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.purple, secondary: colors_1.default.fuchsia, accent: colors_1.default.pink, gray: grays_1.grays.slate }),
    oz: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.indigo, secondary: colors_1.default.cyan, accent: colors_1.default.fuchsia, success: colors_1.default.lime, danger: colors_1.default.rose, gray: grays_1.grays.indigray }),
    romance: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.rose, secondary: colors_1.default.red, accent: colors_1.default.orange, gray: grays_1.grays.zinc }),
    energy: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.amber, secondary: colors_1.default.yellow, accent: colors_1.default.lime, gray: grays_1.grays.stone }),
    winter: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.cyan, secondary: colors_1.default.sky, accent: colors_1.default.emerald, danger: colors_1.default.rose, warning: colors_1.default.amber, info: colors_1.default.cyan, gray: grays_1.grays.cyanish }),
    spring: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.lime, secondary: colors_1.default.green, accent: colors_1.default.yellow, gray: grays_1.grays.greenish }),
    nature: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.green, secondary: colors_1.default.lime, accent: colors_1.default.teal, gray: grays_1.grays.greenish }),
    trust: __assign(__assign({}, defaultFeedbackColors_1.defaultFeedbackColors), { primary: colors_1.default.blue, secondary: colors_1.default.purple, accent: colors_1.default.lime, gray: grays_1.grays.slate }),
};
var grays_2 = require("./grays");
Object.defineProperty(exports, "grays", { enumerable: true, get: function () { return grays_2.grays; } });
var defaultFeedbackColors_2 = require("./defaultFeedbackColors");
Object.defineProperty(exports, "defaultFeedbackColors", { enumerable: true, get: function () { return defaultFeedbackColors_2.defaultFeedbackColors; } });
