// @tailus/themer/dist/components/card.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.gradientCard = exports.card = void 0;
var tailwind_variants_1 = require("tailwind-variants");
exports.card = (0, tailwind_variants_1.tv)({
    base: "block p-[--card-padding] rounded-[--card-radius] bg-[--ui-bg]",
    variants: {
        variant: {
            outlined: "border",
            elevated: "card-shadow",
            soft: "bg-[--ui-soft-bg]",
            mixed: "border card-shadow",
        },
        fancy: {
            true: "fancy-border",
        },
    },
});
exports.gradientCard = (0, tailwind_variants_1.tv)({
    slots: {
        outer: "bg-gradient-to-b from-[--ui-border-color] to-transparent p-px rounded-[--card-radius]",
        inner: "p-[--card-padding] rounded-[calc(var(--card-radius)-1px)]",
    },
    variants: {
        variant: {
            soft: {
                inner: "bg-[--ui-soft-bg]",
            },
            elevated: {
                inner: "bg-[--ui-bg] card-shadow",
            },
        },
    },
});
