// @tailus/themer/dist/components/separator.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.separator = void 0;
var tailwind_variants_1 = require("tailwind-variants");
exports.separator = (0, tailwind_variants_1.tv)({
    base: "bg-[--ui-border-color]",
    variants: {
        fancy: {
            true: "bg-white dark:bg-[--ui-fancy-bg] dark:border-[--ui-fancy-border-color]",
        },
        dashed: {
            true: "border-dashed bg-transparent dark:bg-transparent",
        },
        orientation: {
            vertical: "w-px h-full",
            horizontal: "h-px w-full",
        },
    },
    compoundVariants: [
        {
            dashed: true,
            orientation: "vertical",
            class: "border-r",
        },
        {
            dashed: true,
            orientation: "horizontal",
            class: "border-b",
        },
        {
            fancy: true,
            orientation: "vertical",
            class: "border-r w-0.5",
        },
        {
            fancy: true,
            orientation: "horizontal",
            class: "border-b h-0.5",
        },
    ],
    defaultVariants: {
        fancy: false,
        orientation: "horizontal",
    },
});
