// @tailus/themer-plugins/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.defaultFeedbackColors = exports.grays = exports.palettes = exports.visualizations = exports.rounded = exports.shade = exports.components = exports.animations = exports.palettesArray = exports.palette = void 0;
var palette_1 = require("./palette");
Object.defineProperty(exports, "palette", { enumerable: true, get: function () { return palette_1.palette; } });
Object.defineProperty(exports, "palettesArray", { enumerable: true, get: function () { return palette_1.palettesArray; } });
var animations_1 = require("./animations");
Object.defineProperty(exports, "animations", { enumerable: true, get: function () { return animations_1.animations; } });
var components_1 = require("./components");
Object.defineProperty(exports, "components", { enumerable: true, get: function () { return components_1.components; } });
var shade_1 = require("./shade");
Object.defineProperty(exports, "shade", { enumerable: true, get: function () { return shade_1.shade; } });
var rounded_1 = require("./rounded");
Object.defineProperty(exports, "rounded", { enumerable: true, get: function () { return rounded_1.rounded; } });
var visualizations_1 = require("./visualizations");
Object.defineProperty(exports, "visualizations", { enumerable: true, get: function () { return visualizations_1.visualizations; } });
var palettes_1 = require("./palettes");
Object.defineProperty(exports, "palettes", { enumerable: true, get: function () { return palettes_1.palettes; } });
Object.defineProperty(exports, "grays", { enumerable: true, get: function () { return palettes_1.grays; } });
Object.defineProperty(exports, "defaultFeedbackColors", { enumerable: true, get: function () { return palettes_1.defaultFeedbackColors; } });
